mosviz/loaders/test/sample_test.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
+
+def test_sample():
+    assert(1==1)
```

Comment:
NVM - i'm going to leave this as a placeholder for now.  Real tests need FITS file creation/destruction routines, and i won't be able to get to that today.